test/problem2_spec.rb

```diff
@@ -0,0 +1,42 @@
+
+require 'rspec'
+require '../src/problem2'
+
+describe Problem2 do
+  describe '.largest_two_of' do
+    context 'when is null array' do
```

Comment:
`null` is not something that ruby has, did you mean nil?

Reply:
I've used null because I think that Unit Testing should not consider how the language is built. So i think 'null' is more understable, but as Martin Fowler says, 'yagni'...